x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/cold_start/cold_start.spec.ts

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { first, last, uniq } from 'lodash';
+import moment from 'moment';
+import { apm, timerange } from '@kbn/apm-synthtrace-client';
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import {
+  APIReturnType,
+  APIClientRequestParamsOf,
+} from '@kbn/apm-plugin/public/services/rest/create_call_apm_api';
+import { RecursivePartial } from '@kbn/apm-plugin/typings/common';
+import { isFiniteNumber } from '@kbn/apm-plugin/common/utils/is_finite_number';
+import type { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
+
+type ColdStartRate =
+  APIReturnType<'GET /internal/apm/services/{serviceName}/transactions/charts/coldstart_rate'>;
+
+export const dataConfig = {
+  serviceName: 'synth-go',
+  coldStartTransaction: {
+    name: 'GET /apple 🍎',
+    duration: 1000,
+  },
+  warmStartTransaction: {
+    name: 'GET /banana 🍌',
+    duration: 2000,
+  },
+};
+
+export async function generateData({
```

Comment:
looks like this is only used on this file

```suggestion
async function generateData({
```

Reply:
removed